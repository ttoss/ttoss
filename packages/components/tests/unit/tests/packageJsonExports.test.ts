import pkg from '../../../package.json';

test('package.json exports should export all components', () => {
  const expectedExports = pkg.exports;
  const publishConfigExports = pkg.publishConfig.exports;

  expect(expectedExports).toMatchSnapshot();
  expect(publishConfigExports).toMatchSnapshot();
});
