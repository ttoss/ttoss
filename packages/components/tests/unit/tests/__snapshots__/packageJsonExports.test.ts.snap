// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`package.json exports should export all components 1`] = `
{
  "./Accordion": "./src/components/Accordion/index.ts",
  "./Drawer": "./src/components/Drawer/index.ts",
  "./InstallPwa": "./src/components/InstallPwa/index.ts",
  "./JsonEditor": "./src/components/JsonEditor/index.ts",
  "./JsonView": "./src/components/JsonView/index.ts",
  "./List": "./src/components/List/index.ts",
  "./Markdown": "./src/components/Markdown/index.ts",
  "./Menu": "./src/components/Menu/index.ts",
  "./Modal": "./src/components/Modal/index.ts",
  "./NotificationCard": "./src/components/NotificationCard/index.ts",
  "./Search": "./src/components/Search/index.ts",
  "./Table": "./src/components/Table/index.ts",
  "./Toast": "./src/components/Toast/index.ts",
}
`;

exports[`package.json exports should export all components 2`] = `
{
  "./Accordion": {
    "import": "./dist/esm/Accordion/index.js",
    "types": "./dist/Accordion/index.d.ts",
  },
  "./Drawer": {
    "import": "./dist/esm/Drawer/index.js",
    "types": "./dist/Drawer/index.d.ts",
  },
  "./InstallPwa": {
    "import": "./dist/esm/InstallPwa/index.js",
    "types": "./dist/InstallPwa/index.d.ts",
  },
  "./JsonEditor": {
    "import": "./dist/esm/JsonEditor/index.js",
    "types": "./dist/JsonEditor/index.d.ts",
  },
  "./JsonView": {
    "import": "./dist/esm/JsonView/index.js",
    "types": "./dist/JsonView/index.d.ts",
  },
  "./List": {
    "import": "./dist/esm/List/index.js",
    "types": "./dist/List/index.d.ts",
  },
  "./Markdown": {
    "import": "./dist/esm/Markdown/index.js",
    "types": "./dist/Markdown/index.d.ts",
  },
  "./Menu": {
    "import": "./dist/esm/Menu/index.js",
    "types": "./dist/Menu/index.d.ts",
  },
  "./Modal": {
    "import": "./dist/esm/Modal/index.js",
    "types": "./dist/Modal/index.d.ts",
  },
  "./NotificationCard": {
    "import": "./dist/esm/NotificationCard/index.js",
    "types": "./dist/NotificationCard/index.d.ts",
  },
  "./Search": {
    "import": "./dist/esm/Search/index.js",
    "types": "./dist/Search/index.d.ts",
  },
  "./Table": {
    "import": "./dist/esm/Table/index.js",
    "types": "./dist/Table/index.d.ts",
  },
  "./Toast": {
    "import": "./dist/esm/Toast/index.js",
    "types": "./dist/Toast/index.d.ts",
  },
}
`;
